neuralprophet/forecaster.py

```diff
@@ -212,7 +212,15 @@ class NeuralProphet:
         COMMENT
         Uncertainty Estimation
         COMMENT
-        quantiles: list, default [0.5]
+        uncertainty_method : str, default ``auto``
+            Specifies the type of uncertainty estimation technique that is being deployed 
+        
+            Options
+                * (default) ``auto``: Automatically infers the uncertainty estimation technique based on the prediction interval or quantiles params.  
+                * ``quantile_regression``: Requires the quantiles to be specified.
+        prediction_interval : float, default None 
+            Width of the uncertainty or confidence intervals provided for the forecast. Must be between (0, 1).
+        quantiles : list, default None
```

Comment:
We could we add a mini example, similar to:
 https://github.com/ourownstory/neural_prophet/pull/746/files